tests/unit/test_getPath_and_moveObject.cpp

```diff
@@ -0,0 +1,52 @@
+#include <H5File.hpp>
+#include <H5Group.hpp>
+#include <H5DataSet.hpp>
+#include <H5DataSpace.hpp>
+#include <H5Attribute.hpp>
+#include <H5PropertyList.hpp>
+
+using namespace HighFive;
+
+int main(int argc, char *argv[])
+{
+    QCoreApplication a(argc, argv);
```

Comment:
Seems Qt specific?

See the other tests in the directory and `CMakeLists.txt` for a better testing structure.

Reply:
I just googled what is C++ tests... it is to comlicated for me now. Have not done this before. Also I don't know CMake, I use QMake (I'm beginner).
So I edited the example and put it to the [/src/examples](https://github.com/KerimMatlab/HighFive/tree/master/src/examples) folder. Here is this [example](https://github.com/KerimMatlab/HighFive/blob/master/src/examples/renaming_and_moving_objects.cpp). I hope it is clear and clean enough
I'm sorry I can't do tests now